lib/matplotlib/patches.py

```diff
@@ -3505,11 +3513,13 @@ class CurveFilledB(_Curve):
 
         def __init__(self, head_length=.4, head_width=.2):
             """
-            *head_length*
-              length of the arrow head
+            Parameters
+            ----------
+            head_length : float, optional, default : .4
+                        Length of the arrow head
 
-            *head_width*
-              width of the arrow head
+            head_width : float, optional, default : .2
```

Comment:
For all of these, the numbers need a leading zero before the decimal (e.g., ".2" -> "0.2")